### ISSUE TITLE
[Feature Request] In VR-Mode render a separate 2D-POV view on the playfield screen instead of the VR-Preview.

### ISSUE THREAD

**Issue by Duglim** [vpinball/vpinball#1863]
_Disclaimer: This may be a stupid idea and technically not feasibel. Close this request, if so. :)_ 

**Request:** 
While playing in VR-Mode, there could be a separate fixed 2D-POV view of the game displayed on the playfield screen. This could be instead or in addition to the current VR-Preview.

**Scenario:**
- Some times you have friends around for a pinball party. All are usually looking at the playfield and cheer up the player.
- Now VR/Quest3 is the "new" thing, but only the player himself is in VR. People around the player can only see the VR-Preview window and it is horrible to follow the gameplay, as it is moving around.

**Idea:** 
- In addition to the 2 VR render views (left eye, right eye), there is a 3rd render view like the normal 2D mode on the PC screen/Playfield screen, that uses adjustable POV settings.
- People watching would have the same view as looking at a 2D cabinet/desktop, while the player has the full VR experience. 
- This 3rd view would NOT need 4K resolution and 30/60 fps would be suficcient, since people only watch and do not play. This could keep the PC/GraphicsCard requirements low enough to still have great VR performance.

Thx and greetings, 
Duglim

**Comment by gjupp** [comment]
That's a pretty cool idea Duglim, no doubt it would need a bit of GPU horsepower but options are always good.
The other advantage is that you could stream the 2D gameplay to remote viewers via scorbit or other methods.

**Comment by SuperFromND** [comment]
I've been thinking of streaming VPX VR stuff on twitch, and this would be a godsend for that, even if it means taking a hit in GPU usage.

**Comment by jakereilley** [comment]
Would love you see this feature. Currently working with a team that is building pinball cabinets that work with a vr headset. Biggest complaint I get is that this isn't a feature.

**Comment by thsa** [comment]
I am dreaming of this since a long time. This way one could just add a VR-headset to a cabinet and keep the social experience with friends...  or play in VR with any controller and mirror the playfield on an additional portrait monitor (or beamer) for spectators

**Comment by matgenois** [comment]
I would really love this one too!! I do too have pinball party and we would like to play in VR and other see the playfield in a 2d  POV view. Thank you guys :)

**Comment by bbecausereasonss** [comment]
I love this idea.

**Comment by francisdb** [comment]
On the Quest you can stream one of the eyes directly to a TV using chromecast. I wonder how involved that would be to implement.

**Comment by Pookamoondo** [comment]
This would be a brilliant feature, as when my friend and I take turns in VR, the other watches on my big 55 inch TV.  Unfortunately, my friend gets very ill when watching me play.  Being able to display the desktop mode view on the TV would allow us to play in VR again as every time he is over, I need to switch back to 2D to play se he can spectate.

**Comment by Kaszi46** [comment]
Great idea!

**Comment by tgrosinger** [comment]
Agreed, this would go such a long way toward making VR pinball a more social activity!

**Comment by firebrand007** [comment]
That's also something I've been wanting for a long while... it'd be an awesome feature, very useful to be able to share your experience !

**Comment by Duglim** [comment]
@vbousquet , do you think this is technically possible?

**Comment by GhostieVR** [comment]
Yes!! This would be an extremely useful feature.

**Comment by gjupp** [comment]
Some progress on this feature it seems. https://www.reddit.com/r/virtualpinball/comments/1tzssno/i_think_streaming_in_vr_is_about_to_get_a_lot/

**Comment by gjupp** [comment]
Basti has been working on this feature and it is now included as part of his modified 10.8 VR Launcher edition.

https://vpuniverse.com/files/file/20108-vpx-launcher-64bit/

You will need to autoupdate to the latest version after installing to see the feature as I dont think its included in the vpuniverse download as yet.

Plenty of other new features and optimisations in his version including a better passthrough mode for all VR tables.

**Comment by Duglim** [comment]
Thx for this great news. Time to Check this out!
